### PR TITLE
SNC: 27 April

### PR DESCRIPTION
**Pull request by Northmoc**
Bunch of cards that don't need engine changes

### CODE REVIEW

**Review by Agetian — APPROVED** [review]

File: celestial_regulator.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Celestial Regulator
+ManaCost:1 W U
+Types:Creature Angel Advisor
+PT:2/3
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigTap | TriggerDescription$ When CARDNAME enters the battlefield, choose target creature you don't control and tap it. If you control a creature with a counter on it, the chosen creature doesn't untap during its controller's next untap step.
+SVar:TrigTap:DB$ Tap | ValidTgts$ Creature.YouDontCtrl | TgtPrompt$ Choose target creature you don't control | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | ConditionPresent$ Creature.YouCtrl+HasCounters | Defined$ Targeted | KW$ HIDDEN CARDNAME doesn't untap during your untap step. | Duration$ Permanent
```
Comment (tool4ever) [inline celestial_regulator.txt:8]: Shouldn't this use `This card doesn't untap during your next untap step.`?

File: forge-gui/res/cardsfolder/upcoming/brass_knuckles.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Brass Knuckles
+ManaCost:4
+Types:Artifact Equipment
+T:Mode$ SpellCast | ValidCard$ Card.Self | Execute$ TrigCopy | TriggerDescription$ When you cast this spell, copy it. (The copy becomes a token.)
+SVar:TrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ Double Strike | CheckSVar$ X | SVarCompare$ GE2 | Description$ CARDNAME has double strike as long as two or more Equipment are attached to it.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/brass_knuckles.txt:6]: can probably be done with IsPresent

**Review by Hanmac — APPROVED** [review]
👍

File: forge-gui/res/cardsfolder/upcoming/celestial_regulator.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Celestial Regulator
+ManaCost:1 W U
+Types:Creature Angel Advisor
+PT:2/3
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigTap | TriggerDescription$ When CARDNAME enters the battlefield, choose target creature you don't control and tap it. If you control a creature with a counter on it, the chosen creature doesn't untap during its controller's next untap step.
+SVar:TrigTap:DB$ Tap | ValidTgts$ Creature.YouDontCtrl | TgtPrompt$ Choose target creature you don't control | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | ConditionPresent$ Creature.YouCtrl+HasCounters | Defined$ Targeted | KW$ HIDDEN This card doesn't untap during your untap step. | Duration$ Permanent
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/celestial_regulator.txt:8]: your **next**

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/celestial_regulator.txt:8]: 🤦‍♂️

**Review by tool4ever — APPROVED** [review]

**Pull request merged**